CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Unreleased
+
+#### Fixed
+
+- [#999](https://github.com/rails-sqlserver/activerecord-sqlserver-adapter/issues/999) Fixed support for index types
```

Comment:
Changelog should contain the link to the PR, not the issue itself. Also, if the issue is still present in Rails 6.1+, target branch must be `main` and changes should be backported to all previous versions that we still support.

Reply:
Fixed in PR to `main` in https://github.com/rails-sqlserver/activerecord-sqlserver-adapter/pull/1002